src/editor/editor_display.hpp

```diff
@@ -95,6 +104,12 @@ class editor_display : public display
 	 * northern hexes in the map area, false if it's over the southern hexes instead.
 	 */
 	bool help_string_at_top_ = false;
+
+	/** Whether the help text is currently shown. */
+	bool help_string_enabled_ = true;
+
+	/** Current help string. */
```

Comment:
Took a moment working out why this is added now, when the functionality to display the text was already present. I suggest "Copy of the help text so the label can be created when help_string_enabled is toggled."

Reply:
I feel these shouldn't be Doxygen comments in the first place since the member variables are not API but noted.